docs/superpowers/plans/2026-04-01-grafana-dashboard.md

```
@@ -0,0 +1,1635 @@
+  "title": "Process RSS Memory",
+  "datasource": { "type": "prometheus", "uid": "${DS_PROMETHEUS}" },
+  "gridPos": { "h": 8, "w": 12, "x": 12, "y": 181 },
+  "targets": [{ "expr": "process_resident_memory_bytes", "refId": "A", "instant": true }],
+  "fieldConfig": { "defaults": { "unit": "bytes", "thresholds": { "mode": "absolute", "steps": [{ "color": "green", "value": null }] } }, "overrides": [] },
+  "options": { "reduceOptions": { "calcs": ["lastNotNull"] }, "colorMode": "value", "graphMode": "area" }
+}
+```
+
+- [ ] **Step 2: Validate JSON and commit**
+
+Run: `python3 -c "import json; d=json.load(open('docs/grafana-dashboard.json')); total=len(d['panels']); collapsed=[p for p in d['panels'] if p.get('collapsed')]; inner=sum(len(p.get('panels',[])) for p in collapsed); print(f'top-level: {total}, collapsed inner: {inner}, total panels: {total+inner}')" `
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)